doc/base/classes.xml

```diff
@@ -30203,35 +30203,35 @@
 			<argument index="0" name="parent" type="Node">
 			</argument>
 			<description>
-				Return the transform [Transform2D] calculated relatively to the parent of this 2D node.
+				Returns the [Transform2D] transform relative to this node's parent.
```

Comment:
I think we could drop the "transform" in the sentence, "[Transform2D]" already makes it clear that it's about a transform.